Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst

```diff
@@ -0,0 +1,129 @@
+
+Post-Condition
+^^^^^^^^^^^^^^
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+The drives are power off, and the AuxTel Control cabinet is locked from the outside. 
+The AuxTel is under LOTO and is not able to move by any controls.
+
+.. _AuxTel-LOTO-procedure-Procedure-Steps:
+
+Procedure Steps
+^^^^^^^^^^^^^^^
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+To put the AuxTel under LOTO, follow these steps:
+
+#. Post in the Slack channels `#summit-announce <https://lsstc.slack.com/archives/C01P41NUR1R>`__ and `#summit-auxtel <https://lsstc.slack.com/archives/C01K4M6R4AH>`__ that you are going to put AuxTel under LOTO. 
+
+#. Announce your intentions on the radio (channel 3).
+
+#. Verify that no one is working in the dome, on the ground or second floors. 
+
+#. Identify the AT Control cabinet that you need to lock out, shown in the figure. 
+
+.. figure:: ./_static/AT-control-cabinet.png
+    :name: AT-control-cabinet
+
+    AT Control cabinet, inside, and EATON brakers.
+
+#. Inside the AT Control cabinet, identify the three (3) EATON brakers also shown in the figure, tagged from left to right. 
+
+   a. DRV #1 AZ AZ - The two drives for Azimuth motion.
+
+   b. DRV #2 EL M3 - The drives for elevation and tertiary mirror motion.
+
+   c. DRV #3 NR1 NR2 - Motors for the nasmyth rotators 1 and 2. 
+
+#. Switch off all three brakers by flipping the switch down. 
+
+#. Close the cabinet, and secure your lock and tag in the loop underneath the cabinet door handle. 
+
+#. Verify that the AuxTel will not move intentionally by sending a command with the Engineering User Interface (EUI) on the machine located on the first floor. 
```

Comment:
This is kind of picky, but do we have a figure or example to show here? I can imagine that people which are familiar with LOTO might want to use the procedure in the dome, even though they haven't used the EUI before.

Reply:
Figure added and clarified: 👍